ethclient/ethclient_test.go

```diff
@@ -171,7 +171,7 @@ func newTestBackend(t *testing.T, config *node.Config, enableHistoricalState boo
 	var chainLength int
 	if enableHistoricalState {
 		actualGenesis = genesisForHistorical
-		chainLength = 10
+		chainLength = 4
```

Comment:
Flagging that I'm not sure if this makes sense or not. It's enough to get around a failing test `TestEthClientHistoricalBackend`. I suspect @protolambda or @sebastianst may be able to give me a hint?

Reply:
Hmm I think it is a new problem, actually. Seeing it on a new branch where I rebased on Seb's latest work.

Reply:
I figured it out, new PR is inbound.